03-Mains/Meridional_heat_transport.ipynb

```diff
@@ -6,45 +6,49 @@
    "source": [
```

Comment:
<p>something's not working here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/634/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>